pyproject.toml

```diff
@@ -248,8 +247,7 @@ tox-gh-actions = "^2.10.0"
 boto3-stubs = "^1.26.71"
 botocore-stubs = "^1.29.71"
 freezegun = "~1.2"
-Jinja2 = "2.11.3"
-markupsafe = "2.0.1"
```

Comment:
We had `markupsafe` as a dependency here, just to pin the version, we don't actually rely on it directly. So we are able to drop it now that we can update flask & werkzeug.